R/html-tweak.R

```diff
@@ -571,6 +570,58 @@ tweak_404 <- function(html, pkg = pkg) {
   TRUE
 }
 
+# Add syntax highlighting to reference sections
+tweak_reference_topic_html <- function(html, pkg = pkg) {
+  # blocks with language information = r
+  r_blocks <- xml2::xml_find_all(html, "//div[contains(@class, 'sourceCode r')]/pre/code")
+  # now add blocks with no language information
+  nolang_r_blocks <- xml2::xml_find_all(html, "//div[contains(@class, 'ref-section')]/pre/code")
```

Comment:
Why do non-R code blocks get class `ref-section`?

Reply:
They do not have this class, it is the class of the parent div (coming from https://github.com/r-lib/pkgdown/blob/ce9781a15c7ea07df9fb17a11295ba4abec0b54b/inst/templates/BS4/content-reference-topic.html#L35).

I am also realizing this exact hierarchy might be broken by #1706. (I have no idea whether it breaks it, it could as it tweaks the same divs).